src/mapml.js

```diff
@@ -1336,10 +1577,13 @@ M.MapMLLayer = L.Layer.extend({
         }
         function _processInitialExtent(content) {
             var mapml = this.responseXML || content;
+            if(!this.responseXML && this.responseText) mapml = new DOMParser().parseFromString(this.responseText,'text/xml');
             if (this.readyState === this.DONE && mapml.querySelector) {
-                var serverExtent = mapml.querySelector('extent'),
+                var serverExtent = mapml.querySelector('extent') || mapml.querySelector('meta[name=projection]'),
```

Comment:
The || here is causing the subsequent application of _synthesizeExtent where there is a `<meta[name=projection]>`, which probably is an error, because we would want to use the *other*  meta elements (as well as this one, granted) to synthesize an extent.  I appreciate that the logic of this section is not really clear; we should discuss and maybe we can clear up that logic so it reads better.

Reply:
I believe this was to account for the removal of unutilized `<extent>` (i.e. `<extent units="CBMTILE"></extent>`), prior to this PR some layers contained an extent element with nothing inside, this was replaced with `<meta name="projection" content="CBMTILE">`, so rather than querying for the empty extent to get the projection (formerly `units="CBMTILE"`), those layers now query for the `meta name="projection"` element, a discussion about the logic would be good